{{ cookiecutter.project_slug }}/devops/README-GITLAB.md

```diff
@@ -0,0 +1,58 @@
+# DevOps pipelines in GitLab CI/CD 🚀
+
+Welcome to the DevOps pipelines guide for deploying your project using GitLab pipelines!
+This README provides step-by-step instructions to set up your GitLab repository and initiate manual deployment pipelines.
+Follow each step carefully to correctly configure your environment and secrets.
+
+## Repository setup 🛠️
+
+See [Get started with GitLab CI/CD](https://docs.gitlab.com/ee/ci/).
+
+### Step 1: Add the deployment environment variables
+
+1.  Visit [GitLab](https://gitlab.com/), or your organization's GitLab instance, and log in with your credentials.
+2.  Go to your repository at [{{ cookiecutter.github_organization }}/{{ cookiecutter.project_slug }}](https://gitlab.com/{{ cookiecutter.github_organization }}/{{ cookiecutter.project_slug }}).
+3.  Click on `Settings` at the left menu and select `CI/CD`.
+4.  Expand the `Variables`.
+5.  Add all variables and their corresponding values, copying them from the `.env` file.
+    This file is not commited into the repository, and that's why we need to add them here.
+
+### Step 2: Add deployment's host SSH key as a known host
+
+1.  Run `ssh-keyscan -t ed25519 DEPLOY_HOST` in a terminal, where you substitute `DEPLOY_HOST` with the address of the host, as in the `.env` file.
+2.  Copy the prompted value, and add it as a `Variable` in the GitLab web interface, like in the previous step.
+    Name the variable `SSH_KNOWN_HOSTS`.
+
+See [Use SSH keys to communicate with GitLab](https://docs.gitlab.com/ee/user/ssh.html).
+
+### Step 3: Add an SSH deployment key
+
+1.  Create a new SSH key that will be used by GitLab CI/CD to connect to your host.
+    You can create it in your terminal with the command `ssh-keygen -t ed25519`.
+2.  Copy the public key to the `DEPLOY_HOST`'s user's `~/.ssh/authorized_keys` file.
+    Check your `DEPLOY_HOST` and `DEPLOY_USER` in the `.env` file.
+3.  Copy the private key as a `Variable` like in the previous step.
+    Name the variable `DEPLOY_SSH_PRIVATE_KEY`.
+
+*Note*: instead of creating a new key, you can use the one that this template creates for you when you run `make server-setup`. Check README.md file for details. Be aware that the key generated by this template will not be added to your git repository because it is explicitely excluded by the `.gitignore` file.
```

Comment:
Please verify the location of `README.md` and section name. I think that's what you meant.

```suggestion
*Note:* instead of creating a new key, you can use the one that this template creates for you when you run `make server-setup`.
Check the section "Server Setup" in the file `README.md` in the same folder as this file for details.
Be aware that the key generated by this template will not be added to your git repository because it is explicitly excluded by the `.gitignore` file.
```

Reply:
Yes, that what I meant